lib/vector-buffers/src/variants/disk_v1/tests/acknowledgements.rs

```diff
@@ -316,7 +316,7 @@ async fn acking_when_undecodable_records_present() {
         });
 
         let parent = trace_span!("acking_when_undecodable_records_present");
-        fut.instrument(parent).await;
+        fut.instrument(parent.or_current()).await;
```

Comment:
Most of these are superfluous, I just wanted to get in the habit of always doing `.or_current()` to avoid forgetting it when it actually matters.